src/cli/commands/version.js

```diff
@@ -146,14 +144,16 @@ export async function setVersion(
       const flag = sign ? '-sm' : '-am';
       const prefix: string = String(config.getOption('version-tag-prefix'));
 
+      const gitRoot = (await spawnGit(['rev-parse', '--show-toplevel'])).trim();
```

Comment:
You probably want to use `{cwd: config.cwd}` so that if we add tests, it will still work.